src/docs/markdown/command-line.md

```diff
@@ -495,6 +503,51 @@ The address of this request can be customized using the `--address` flag, or fro
 If you want to stop the current configuration but do not want to exit the process, use [`caddy reload`](#caddy-reload) with a blank config, or the [`DELETE /config/`](/docs/api#delete-configpath) endpoint.
 
 
+### `caddy storage`
+
+Allows export and import of the contents of Caddy's configured data storage.
```

Comment:
Should we say this is still experimental (liberty to change it)?

Reply:
Oh right, sure.